src/util/completeFromCT.js

```diff
@@ -0,0 +1,18 @@
+const fn = require('../config/fieldNames');
+
+/**
+ * This module add fields from city dataSource to given object
+ *
+ * @param {*} obj Object of person
+ * @param {*} CTRecord city dataSource
+ * @returns Object of person with the data from city
+ */
+module.exports =  (obj, CTRecord) => {
+    
+    obj.rank = CTRecord[fn.city.rank];
```

Comment:
add identityCard currentUnit and every other possible field - like aka

Reply:
aka: {
        serviceType: "nstype",
        firstName: "firstName",
        lastName: "lastName",
        identityCard: "tz",
        personalNumber: "mi",
        rank: "rnk",
        phone: "telephone",
        areaCode: "ktelephone",
        mobilePhone: "telephone",
        areaCodeMobile: "ktelephone",
        dischargeDay: "rld",
        clearance: "clearance",
        unitName: "hr", 
        telephoneType: "telephoneType",
        uniqeFieldForDeepDiff: "mi",
        birthday: "birthday",
        sex: "sex",
        picture: "picture",
    },

 city_name: {
        firstName: 'firstName',
        lastName: 'lastName',
        identityCard: 'tz',
        rank: 'rank',
        dischargeDay: 'rld',
        clearance: 'clearance',
        currentUnit: 'department',
        serviceType: 'stype',
        mobilePhone: 'telephone',
        personalNumber: 'personalNumber',
        mail: 'mail',
        job: 'job',
        profession: 'profession',
        hierarchy: 'hr',
        domainUsers: 'domUser',
        company: 'company',
        entityTypePrefix: {
            s: ['e', 'l', 'i'],
            c: ['g', 'j'],
            gu: ['m'],
        },

which ones to be exact? first name and last name too for example?